share/locale/en.json

```diff
@@ -12,6 +12,7 @@
     "gui_add_files": "Add Files",
     "gui_add_folder": "Add Folder",
     "gui_delete": "Delete",
+    "gui_file_selection_clear_all": "Clear All",
```

Comment:
Given that we use the word "Delete" to remove each file from the UI, should we use "Delete All" instead of "Clear All" to make it clear that both have the same effect?

Reply:
Yeah, this was tricky, because we use 'Clear All' in the toggled History view, for history items. I originally had 'Delete' and then I thought it might be better to make it consistent with the History 'Clear All' button.

But, I think I agree with you: from a UX perspective, 'Delete' is more consistent with the 'Delete' button for individual files, and users probably associate 'Delete' with files and 'Clear' with 'history' (like 'clearing' browser history). So I'm changing it back :) Thanks for the review.

Reply:
Honestly I feel "Remove" makes the intent clear the best. Because it's clear you are removing the files from share list. Delete sometimes might feel like delete from where. But given that people are already familiar with what Delete means in respect to onionshare, I think this is fine.